### DISCUSSION TITLE
will there be an android app in the future?

### DISCUSSION THREAD

**Discussion by mo-han** [TriliumNext/Trilium#2751] [Q&A]
:)

**Comment by zadam** [comment]
Most probably no :-(

**Reply by mo-han** [reply]
but we definitely want trilium on mobile, we all do.
I really hope I'm a skillful android app developer, but I couldn't help with this.
sad.

**Reply by zadam** [reply]
There's a "mobile frontend" which works better with small screens than the standard interface:

https://github.com/zadam/trilium/wiki/Mobile-frontend

But it's not full featured and obv. isn't a real mobile app.
